src/Sulu/Component/Rest/ListBuilder/Expression/Doctrine/DoctrineBetweenExpression.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Expression\Doctrine;
+
+use Doctrine\ORM\QueryBuilder;
+use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\AbstractDoctrineFieldDescriptor;
+use Sulu\Component\Rest\ListBuilder\Expression\BetweenExpressionInterface;
+
+/**
+ * Represents a BETWEEN expression for doctrine - needs a field and two values.
+ */
+class DoctrineBetweenExpression extends AbstractDoctrineExpression implements BetweenExpressionInterface
+{
+    /**
+     * Field descriptor used for comparison.
+     *
+     * @var AbstractDoctrineFieldDescriptor
+     */
+    protected $field;
+
+    /**
+     * @var
+     */
+    protected $start;
+
+    /**
+     * @var
```

Comment:
I don't know the type at this point. Could be anything. All I can add is `mixed`.